lightwood/mixer/sktime.py

```diff
@@ -44,8 +76,11 @@ def fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
                                for gcol in self.grouped_by} if self.ts_analysis['tss'].group_by else {}}
 
         for group in self.ts_analysis['group_combinations']:
-            # many warnings might be thrown inside of statsmodels during stepwise procedure
-            self.models[group] = self.model_class(suppress_warnings=True)
+            # ignore warnings if possible
```

Comment:
might want to make suppress warnings optional with the default true

Reply:
Will leave it as non-configurable for now (warnings are really verbose, we can reactivate this is users want it).